code/go/0chain.net/sharder/m_handler.go

```diff
@@ -153,7 +153,7 @@ func (sc *Chain) cacheProcessingBlock(hash string) bool {
 	_, err := sc.processingBlocks.Get(hash)
 	switch err {
 	case cache.ErrKeyNotFound:
-		if err := sc.processingBlocks.Add(hash, struct{}{}); err != nil {
+		if err := sc.processingBlocks.Add(hash, &block.Block{}); err != nil {
```

Comment:
Let's keep using the`struct{}{}`, and define the `processingBlocks` as `cache.LRU[string, struct{}]`.

Reply:
@peterlimg why we should do it, if they are blocks?

Reply:
it's the same as why do `map[sting]stuct{}` instead of `map[string]*block.Block`.